### DISCUSSION TITLE
SSL for localhost

### DISCUSSION THREAD

**Discussion by StorytellerCZ** [meteor/meteor#11996] [Feature Requests]
While working on Web3 login package I needed to have my localhost under https to be able to work with it. After some digging I used `nourharidy:ssl` with some updates on my and @harryadel part. It would have a simplified functionality for this in core similar like [parcel has](https://parceljs.org/features/development/#https).

Right now the `nourharidy:ssl` requires you to manually create the certificate, so if we could automate that part, that would be awesome.

Thoughts?

**Comment by harryadel** [comment]
It still has a niche use when developing security related features so I'm a not fan of adding it the core, *yet*. You can actually publish a new package with your modifications and see how it goes from there.
